roles/migrationcontroller/templates/velero_supporting.yml.j2

```diff
@@ -15,7 +15,7 @@ spec:
     singular: backup
   preserveUnknownFields: false
   scope: Namespaced
-{% if lookup('k8s', cluster_info='version').kubernetes.minor|replace('+', '')|int < 11 %}
+{% if lookup('k8s', cluster_info='version').kubernetes.minor|replace('+', '')|int > 10 %}
```

Comment:
Is this a purposeful change or got pulled in by mistake?

Reply:
Pulled by mistake. Should only be the readme. Probably need to rebase.